### PR TITLE
refactor: remove jinad

### PR DESCRIPTION
**Pull request by jacobowitz**
# Remove JinaD

## Description

This PR removes JinaD. This should include all code, tests and CI integrations.

### CODE REVIEW

**Comment by deepankarm** [comment]
image

File: Dockerfiles/debianx.Dockerfile [hunk 1]
```diff
@@ -71,13 +71,4 @@ RUN cd /jina && \
 
 ENTRYPOINT ["jina"]
 
-FROM jina AS jina_daemon
```
Comment (JoanFM) [inline Dockerfiles/debianx.Dockerfile:74]: then maybe we can remove this file?

Comment (jacobowitz) [reply Dockerfiles/debianx.Dockerfile:74]: we are still using it in CI. I am not completely sure what is the difference between this file and the PIP one. But we are using this e.g. in the ARM64 test I believe. Maybe for other things as well.

Comment (jacobowitz) [reply Dockerfiles/debianx.Dockerfile:74]: e.g. [here](https://github.com/jina-ai/jina/blob/master/.github/workflows/ci.yml#L142)

**Review by JoanFM — APPROVED** [review]

**Pull request merged**